### ISSUE TITLE
Setup fails to test for helm as non root macOS user

### ISSUE THREAD

**Issue by pinheadmz** [bitcoin-dev-project/warnet#661]
In this environment the subprocess module raises a permission error not a FileNotFound error. So it isn't caught by the try catch.

**Comment by mplsgrant** [comment]
Is this #640 -> #654?

**Comment by pinheadmz** [comment]
I dont think so, more like replacing all `FileNotFoundError` with plain `Exception` in project.py

**Comment by Anyitechs** [comment]
Hi @pinheadmz, I'll like to work on this issue. Can you assign it to me?

**Comment by willcl-ark** [comment]
Hi @Anyitechs, we don't really work on assigning basis here, but please feel free to pick up the changes here (I assigned it to you anyway 😄 )

**Comment by Anyitechs** [comment]
> Hi @Anyitechs, we don't really work on assigning basis here, but please feel free to pick up the changes here (I assigned it to you anyway 😄 )

Thank you. I'll send in a draft PR shortly for concept ACK.
